test/integration/targets/delegate_to/aliases

```diff
@@ -2,3 +2,4 @@ shippable/posix/group3
 needs/ssh
 needs/root  # only on macOS and FreeBSD to configure network interfaces
 skip/aix
+non_local # workaround to allow override of ansible_python_interpreter; disables coverage on this integration target
```

Comment:
@mkrizek  The `non_local` entry should not be added to existing tests, since it disables coverage support for those tests. If it is needed for the test to work, a new test needs to be created.

When you're ready to work on a revised version of this, I'd like to discuss the tests to see if we can avoid the use of `non_local`.